docs/src/examples/components/Popup/Types/PopupControlledExample.shorthand.tsx

```diff
@@ -1,24 +1,27 @@
+import { useBooleanKnob } from '@stardust-ui/docs-components'
 import * as React from 'react'
-import { Button, Input, Popup } from '@stardust-ui/react'
+import { Button, Input, Popup, Flex } from '@stardust-ui/react'
 
-class PopupControlledExample extends React.Component {
-  state = { open: false }
+const PopupControlledExample = () => {
+  const [open, setOpen] = useBooleanKnob({ name: 'open' })
 
-  handleOpenChange = (e, { open }) => {
-    alert(`Popup requested to change its open state to "${open}".`)
-    this.setState({ open })
-  }
+  const popupContent = (
+    <Flex column>
+      <Flex.Item align="end">
+        <Button text iconOnly icon="close" onClick={() => setOpen(false)} />
```

Comment:
please add `title='Close'`

Reply:
also, not directly related to the PR but since the popup contains actionable components, it should have `trapFocus`.

Reply:
Will add the `trapFocus` prop. Also added title on the trigger button as it only contains an icon